design-tokens/tokens/layer2/dark.json

```diff
@@ -305,6 +305,21 @@
               "type": "color"
             }
           }
+        },
+        "backDrop": {
+          "default": {
+            "value": "{echoes.color.grey.900}",
```

Comment:
It's the same backdrop for light and dark theme?

Reply:
Fixed ✅